transcom/mymove
# B 21163 transportation offices ak hi

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1011472&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## [INT PR](https://github.com/transcom/mymove/pull/13852)

## Summary
This is to add the new AK and HI transportation offices to the DB, along with the associated tables.  Spreadsheet is attached to the Feature, came down from J9.

### How to test

1. Check the db tables to see that the new data populated.
2. Login as an Admin user, create or edit a office user and verify that you see the new Transportation offices in the transportation office dropdown.
3. Pick a new transportation office for AK and save to an office user.  Login as that office user and verify you see the corresponding GBLOC in the UI for that user.
4. Repeat step 3 for a HI transportation office.
